### DISCUSSION TITLE
Waku Meaning

### DISCUSSION THREAD

**Discussion by anaclumos** [wakujs/waku#260] [General]
Out of curiosity, what does Waku mean?

**Comment by dai-shi** [comment]
In Japanese, it's frame or framework. https://www.romajidesu.com/dictionary/meaning-of-waku.html

It's also famous for Anya's "Waku-Waku". https://shotora.net/waku-waku-meaning/

**Reply by himself65** [reply]
![image](https://github.com/dai-shi/waku/assets/14026360/d23728cb-f3d9-4086-9937-21fa7842e041)
